CHANGELOG.MD

```diff
@@ -28,14 +28,37 @@
 
 ### Formatter
 
-- Set default of CheckParameter configuration setting to false to ensure that setting won't be enabled by default if an old version of the VS-Code extension uses a new version of - PSScriptAnalyzer (1.19) (#1411) (by @bergmeister)
-- Split CheckPipe feature of trimming redundant whitespace out into option CheckPipeForRedundantWhiteSpace (#1413) (by @bergmeister)
-- Fix recently introduced bug (PR #1391) for UseCorrectCasing (#1408) (by @bergmeister)
-- Add PipelineIndentationStyle.None option for scenarios where indentation is custom, inconsistent or the user does not like any of the 3 pipeline indentation styles (#1399) (by @bergmeister)
-- Correct casing of parameters as well in PSUseCorrectCasing formatter rule (#1391) (by @bergmeister)
-- Remove redundant whitespace between parameters with new option (disabled by default) in UseConsistentWhitespace (#1392) (by @bergmeister)
-- Fix complex case when PipelineIndentation is not set to default (NoIndentation) (#1359) (by @bergmeister)
-- UseConsistentWhitespace: Ignore empty hashtable for CheckInnerBrace configuration (#1349) (by @bergmeister)
+- UseCorrectCasing
+  - Correct casing of parameters as well in PSUseCorrectCasing formatter rule (#1391) (by @bergmeister)
+  - Fix temporarilly introduced bug (PR #1391) for UseCorrectCasing (#1408) (by @bergmeister)
+- UseConsistentIndentation
+  - When non-default options are used, cater for the case of a comment between pipe and newline (#1463)
+  - Add PipelineIndentationStyle.None option for scenarios where indentation is custom, inconsistent or the user does not like any of the 3 pipeline indentation styles (#1399) (by @bergmeister)
+  - Fix complex case when PipelineIndentation is not set to default (NoIndentation) (#1359) (by @bergmeister)
+- UseConsistentWhitespace
```

Comment:
It looks like this switches from point on every second line to point on every line here -- is that intentional?

Reply:
No, will fix that

Reply:
No, will fix that

Reply:
actually no, it just looks like that in GitHub's diff